Main.sublime-menu

```diff
@@ -208,14 +208,10 @@
         ]
     },
     {
-        "caption": "Preferences",
```

Comment:
You are going to have to put BracketHighlighter back under `"id": "packages"` before I consider merging this.  I don't mind merging the other stuff.